packages/shared/blockchain/evm-based/src/lib/transactions/custodial/index.ts

```diff
@@ -35,6 +46,95 @@ const generateCustodialWallet = async (
   )
 }
 
+const transferFromCustodialWallet = async (
+  body: ChainTransferCustodialWallet,
+  web3: EvmBasedWeb3,
+  testnet?: boolean,
+  provider?: string,
+) => {
+  return evmBasedCustodial().prepareTransferFromCustodialWalletAbstract(
+    body,
+    web3,
+    evmBasedUtils.decimals,
+    smartContractWriteMethodInvocation,
+    18,
+    testnet,
+    provider,
+  )
+}
+
+const batchTransferFromCustodialWallet = async (
+  body: ChainBatchTransferCustodialWallet,
+  web3: EvmBasedWeb3,
+  testnet?: boolean,
+  provider?: string,
+) => {
+  return evmBasedCustodial().prepareBatchTransferFromCustodialWalletAbstract(
+    body,
+    web3,
+    evmBasedUtils.decimals,
+    smartContractWriteMethodInvocation,
+    18,
+    testnet,
+    provider,
+  )
+}
+
+const getCustodialFactoryContractAddress = (testnet?: boolean) => {
```

Comment:
what are these 2 addresses? for which chain? Each supported chain has different addresses - this should be parametrized at least

Reply:
completely overlooked this, I thought it was from core - will update